deps/chakrashim/lib/chakra_shim.js

```diff
@@ -276,7 +277,7 @@
       Error, EvalError, RangeError, ReferenceError, SyntaxError, TypeError,
       URIError
     ].forEach(function(type) {
-      var newType = function __newType() {
+      var newType = function newType() {
```

Comment:
nit: Why change "__newType" function name? I forgot the exact context/reason of the naming... maybe just to have less chance to be the same as a user function name.